README.md

```diff
@@ -587,6 +587,13 @@ If a location should report the empty string, use the special label `(nothing)`:
 # ^ hover: (nothing)
 ```
 
+Assert the contents of a specific line of the hover response with `hover-line` assertions:
+
+```ruby
+  a = 10
+# ^ hover-line: 0 Integer(10)
```

Comment:
Does it make more sense to be `0` or `1` indexed?

Reply:
I think most editors use 1-based line indexing, so IMO 1-based is what I'd prefer.

Reply:
Awesome, that's where I was leaning too. Will update now.